### ISSUE TITLE
Unable to change URI (only /centreon) works

### ISSUE THREAD

**Issue by BenoitPoulet** [centreon/centreon#7706]
# BUG REPORT INFORMATION

### Prerequisites

***Versions***
```
centreon-plugin-Applications-Protocol-Ftp-20190704-123459.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Poller-20190704-123459.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ldap-20190704-123459.el7.centos.noarch
centreon-plugin-Operatingsystems-Linux-Snmp-20190704-123459.el7.centos.noarch
centreon-clib-19.04.0-1.el7.centos.x86_64
centreon-trap-19.04.2-7.el7.centos.noarch
centreon-poller-centreon-engine-19.04.2-7.el7.centos.noarch
centreon-auto-discovery-server-19.04.0-4.el7.centos.x86_64
centreon-widget-global-health-19.04.0-5.el7.centos.noarch
centreon-widget-graph-monitoring-19.04.0-5.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Central-20190704-123459.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Database-20190704-123459.el7.centos.noarch
centreon-plugin-Network-Cisco-Standard-Snmp-20190704-123459.el7.centos.noarch
centreon-release-19.04-1.el7.centos.noarch
centreon-perl-libs-19.04.2-7.el7.centos.noarch
centreon-connector-perl-19.04.0-2.el7.centos.x86_64
centreon-license-manager-common-19.04.0-1.el7.centos.noarch
centreon-awie-19.04.0-1.el7.centos.noarch
centreon-widget-tactical-overview-19.04.0-7.el7.centos.noarch
centreon-plugin-Applications-Databases-Mysql-20190704-123459.el7.centos.noarch
centreon-broker-core-19.04.0-2.el7.centos.x86_64
centreon-engine-19.04.1-9.el7.centos.x86_64
centreon-database-19.04.2-7.el7.centos.noarch
centreon-widget-grid-map-19.04.0-5.el7.centos.noarch
centreon-fingerprint-19.04.0-1.el7.centos.x86_64
centreon-common-19.04.2-7.el7.centos.noarch
centreon-broker-storage-19.04.0-2.el7.centos.x86_64
centreon-broker-cbmod-19.04.0-2.el7.centos.x86_64
centreon-connector-19.04.0-2.el7.centos.x86_64
centreon-pp-manager-19.04.0-3.el7.centos.noarch
centreon-19.04.2-7.el7.centos.noarch
centreon-widget-hostgroup-monitoring-19.04.0-5.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-19.04.0-5.el7.centos.noarch
centreon-widget-host-monitoring-19.04.0-5.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Map4-Jmx-20190704-123459.el7.centos.noarch
centreon-plugin-Hardware-Printers-Generic-Snmp-20190704-123459.el7.centos.noarch
centreon-engine-daemon-19.04.1-9.el7.centos.x86_64
centreon-broker-cbd-19.04.0-2.el7.centos.x86_64
centreon-web-19.04.2-7.el7.centos.noarch
centreon-widget-engine-status-19.04.0-5.el7.centos.noarch
centreon-widget-live-top10-memory-usage-19.04.0-5.el7.centos.noarch
centreon-plugin-Applications-Protocol-Http-20190704-123459.el7.centos.noarch
centreon-plugin-Applications-Protocol-Dns-20190704-123459.el7.centos.noarch
centreon-plugin-Hardware-Ups-Standard-Rfc1628-Snmp-20190704-123459.el7.centos.noarch
centreon-broker-19.04.0-2.el7.centos.x86_64
centreon-engine-extcommands-19.04.1-9.el7.centos.x86_64
centreon-base-config-centreon-engine-19.04.2-7.el7.centos.noarch
centreon-widget-service-monitoring-19.04.0-5.el7.centos.noarch
centreon-plugin-Operatingsystems-Windows-Snmp-20190704-123459.el7.centos.noarch
centreon-connector-ssh-19.04.0-2.el7.centos.x86_64
centreon-license-manager-19.04.0-1.el7.centos.noarch
centreon-widget-httploader-19.04.0-5.el7.centos.noarch
centreon-widget-servicegroup-monitoring-19.04.0-7.el7.centos.noarch

```
***Operating System***

*CentOS*

***Browser used***

- [ ] Google Chrome
- [*] Firefox
- [ ] Internet Explorer IE11
- [ ] Safari

Version: 68

***Additional environment details (AWS, VirtualBox, physical, etc.):***

### Description

I'm unable to change the URI from "/centreon" to "/"
When i do this, i got a blank page, without any error in Apache logs or PHP logs

### Steps to Reproduce

Please describe precisely the steps to reproduce the encountered issue.

0. in  "Administration  >  Parameters  >  Centreon UI > Centreon Web Directory", change the value to "/"
1. Edit "/opt/rh/httpd24/root/etc/httpd/conf.d/10-centreon.conf"
2. Change "Alias /centreon /usr/share/centreon/www/"  to "Alias / /usr/share/centreon/www/"
3. Change "<LocationMatch ^/centreon/(.*\.php(/.*)?)$>" to "<LocationMatch ^/(.*\.php(/.*)?)$>"
4. Comment "RedirectMatch ^/$ /centreon"
5. Restart Apache systemctl restart httpd24-httpd

### Describe the received result
in the browser, / is redirected to /main.php, then a blank page is shown.

### Describe the expected result
the centreon's homepage.

### Logs

No error, il any log.

**Comment by lpinsivy** [comment]
Hi @BenoitPoulet do you have error in /var/opt/rh/rh-php71/log/php-fpm/centreon-error.log file?

**Comment by BenoitPoulet** [comment]
Hi @lpinsivy 

No error in this file (it remains empty)

**Comment by BenoitPoulet** [comment]
Hello, i would like how i can help with this bug as it's a blocker for my upgrade.

**Comment by BenoitPoulet** [comment]
Hello, 

New year, any plan for a fix ?

**Comment by kduret** [comment]
Hi !
It is fixed with Centreon 20.04.0 : https://github.com/centreon/centreon/commit/a640a39826185a876400e6e09cf093d6dc7e412b
